layout/theme.liquid

```diff
@@ -51,6 +51,46 @@
       {{ body_font_bold_italic | font_face: font_display: 'swap' }}
       {{ settings.type_header_font | font_face: font_display: 'swap' }}
 
+      {% for scheme in settings.color_schemes %}
+        {% if forloop.index == 1 %}
+          :root,
+        {% endif %}
+        .color-{{ scheme.id }} {
+          --color-scheme-background: {{ scheme.settings.background.red }},{{ scheme.settings.background.green }},{{ scheme.settings.background.blue }};
```

Comment:
I added a decision entry for this in the PR description, but this gist is that is the same syntax we're doing in Dawn now so I don't think it's too controversial to keep using it for the time being. I want to use `color.rgb` but it returns its value in a space-sperated format that the legacy css rgba isn't compatible with. So I'd need to update ~300 references of `rgba(0,0,0,0)` to use `rgb(0 0 0 / 0)`. I ultimately decided to not include in this PR because there's enough to look at here already.

Reply:
> One big change from the old colour scheme is that Scheme 1 really does a lot of heavy lifting

This isn't really changed, we're just being transparent about it now. None of our colors are inherently global anymore (only existing within the context of a particular scheme). So it seemed important to clarify that some areas implicitly use a scheme.